DBlock Game/DBlock Game/Program.cs

```diff
@@ -1558,6 +1563,239 @@ public static void loading()
+        * westhallway coneccts the other two hallways. has a couple of places to hide in to increase chances of survival
+        * use the enemyinfo method to give a player an update on where the enemey is.
+        * uses the survival check to see if a plaer lives or dies and resets the enemies location.
+        */
+        public void westHallway_Basement()
+        {
+            enemyLocation = 5;
+            playerLocation = 0;
+            enemyInfo = "You hear a screeching noise coming from behind and a robot saying trepassers will be terminated ";
+            random = new Random();
+
+            loading();
+            TextBox("You have entered the west hall way", Player, false);
+            TextBox(enemyInfo, Player, false);
+            TextBox("You see a couple of lockers and a table that you could possibly hide under", Player, false);
+            TextBox("How would you like to procede??", Player, true);
+                bool fail = false;
+            do
+            {
+                TextBox("What should I do now? \n" + "║1: Go under table \n║2: go in locker  \n║3: Move north hallway \n║4: Move to West Hallway", Player, true);
+                string temp = Console.ReadLine();
+                switch (temp)
+                {
+                    case "1":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation,enemyLocation);
+                        chance = 2;
+                        westHallway_Basement();
+                        break;
+                    case "2":
+                        chance = 5;
+                        enemyMovement(1);
+                        surviveCheck(playerLocation, enemyLocation);
+                        chance = 2;
+                        westHallway_Basement();
+                        break;
+
+                    case "3":
+                        enemyMovement(2);
+                        surviveCheck(playerLocation, enemyLocation);
+                        northHallway_Basement();
+                        break;
+
+                    case "4":
+                        enemyMovement(2);
+                        surviveCheck(playerLocation,enemyLocation);
+                        southHallway_Basement();
+                        break;
+
+                    case "map":
+                        Map(2);
+                        break;
+
+                    case "inv":
+                        Inventory();
+                        break;
+
+                    case "help":
+                        Help();
+                        break;
+
+                    default:
+                        fail = true;
+                        break;
+                } 
+            } while (fail == true);
+        }
+        /**
+        * template for the hallways jus need rooms attached to the case statements
+        */
+        public void northHallway_Basement()
```

Comment:
remember, public methods are capitalised, and private methods are lowercase. so this should be NorthHallway_Basement

Reply:
made changes